imageroot/actions/get-configuration/10get

```diff
@@ -5,13 +5,66 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 #
 
+from datetime import datetime, timezone
+
 import os
 import json
 import sys
+import subprocess
+
+# Cloud Log Manager
+
+cloud_log_manager = {}
+
+with subprocess.Popen(['systemctl', '--user', 'is-active', 'cloud-log-manager-forwarder'], stdout=subprocess.PIPE, stderr=sys.stderr) as pipe:
+    response = pipe.stdout.readline().decode('utf-8').replace('\n', '')
+    if 'inactive' in response:
+        cloud_log_manager["status"] = 'inactive'
+    elif 'active' in response:
+        cloud_log_manager["status"] = 'active'
+    elif 'failed' in response:
+        cloud_log_manager["status"] = 'failed'
+
+cloud_log_manager["address"] = os.getenv('CLOUD_LOG_MANAGER_ADDRESS', '')
+cloud_log_manager["tenant"] = os.getenv('CLOUD_LOG_MANAGER_TENANT', '')
+
+try: 
+    with open("./cloud_log_manager_last_timestamp", "r") as clm_file:
+        cloud_log_manager["last_timestamp"] = clm_file.read()
+except:
+    cloud_log_manager["last_timestamp"] = ""
+
+# Syslog
+
+syslog = {}
+
+with subprocess.Popen(['systemctl', '--user', 'is-active', 'syslog-forwarder'], stdout=subprocess.PIPE, stderr=sys.stderr) as pipe:
+    response = pipe.stdout.readline().decode('utf-8').replace('\n', '')
+    if 'inactive' in response:
+        syslog["status"] = 'inactive'
+    elif 'active' in response:
+        syslog["status"] = 'active'
+    elif 'failed' in response:
+        syslog["status"] = 'failed'
```

Comment:
I see many of `Popen`, it's the constructor used by `subprocess.run`, however use it only when you have to do very fancy things, I'm going to mark all the occurrences so we can remember the code to convert